includes/today-options-gmucf.php

```diff
@@ -18,6 +19,306 @@ function tu_add_gmucf_options_page() {
+									'preview_size'      => 'full',
+									'library'           => 'all',
+									'mime_types'        => 'jpg, jpeg',
+								),
+								array(
+									'key'               => 'field_5bef13d1868d4',
+									'label'             => 'Story Title',
+									'name'              => 'gmucf_story_title',
+									'type'              => 'text',
+									'instructions'      => 'This will replace the story\'s title in the email.',
+								),
+								array(
+									'key'               => 'field_5bef13d1868d5',
+									'label'             => 'Story Description',
+									'name'              => 'gmucf_story_description',
+									'type'              => 'textarea',
+									'instructions'      => 'This will replace the story\'s Deck field value in the email.',
+								),
+							),
+						),
+					),
+					'min'        => '1',
+					'max'        => '',
+				),
+				'5be5fff46ee51' => array(
+					'key'        => '5be5fff46ee51',
+					'name'       => 'gmucf_spotlight',
+					'label'      => 'Spotlight',
+					'display'    => 'block',
+					'sub_fields' => array(
+						array(
+							'key'               => 'field_5be601316ee53',
+							'label'             => 'Spotlight Link',
+							'name'              => 'gmucf_spotlight_link',
+							'type'              => 'url',
+							'instructions'      => 'Enter the URL that users will be directed to when clicking on the spotlight image.',
+							'required'          => 1,
+						),
+						array(
+							'key'               => 'field_5be601706ee54',
+							'label'             => 'Spotlight Alt Text',
+							'name'              => 'gmucf_spotlight_alt_text',
+							'type'              => 'text',
+							'instructions'      => 'Enter the alternative text for the spotlight image here. Describe what the image is of and any text in the image. This text will also be used in the text only version of the email, so make sure it\'s descriptive.',
+							'required'          => 1,
+						),
+						array(
+							'key'               => 'field_5be600be6ee52',
+							'label'             => 'Spotlight Image',
+							'name'              => 'gmucf_spotlight_image',
+							'type'              => 'image',
+							'instructions'      => 'Select an image to use as the spotlight.',
+							'required'          => 1,
+							'preview_size'      => 'full',
+							'mime_types'        => 'jpg, jpeg, gif',
+						),
+					),
+					'min'        => '',
+					'max'        => '2',
+				),
+			),
+			'button_label'      => 'Add Content',
+		);
+
+		// Adds GMUCF Announcements field
+		$gmucf_options_fields[] = array(
+			'key'               => 'field_5c5ca43d3a608',
+			'label'             => 'GMUCF Announcements',
+			'name'              => 'gmucf_announcements',
+			'type'              => 'select',
+			'instructions'      => 'Select the announcements to be displayed on the News and Announcements email.',
+			'default_value'     => array(
```

Comment:
An empty array can just be on a single single for the sake of readability.

Reply:
Updated